### PR TITLE
Fix rand.

### PR DESCRIPTION
**Pull request by shmokmt**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -32,6 +32,5 @@ func main() {
 		"SUM関数",
 	}
 	rand.Seed(time.Now().UnixNano())
-	max := len(somethingList)
-	fmt.Printf("寒すぎて、%sになった\n", somethingList[rand.Intn(max)])
+	fmt.Printf("寒すぎて、%sになった\n", list[rand.Int()%len(list)])
```
Comment (shmokmt) [inline main.go:35]: > Intn returns, as an int, a non-negative pseudo-random number in [0,n) from the default Source. It panics if n <= 0.

So we can't be samus（サムス）.

https://golang.org/pkg/math/rand/#Intn

Comment (shogo82148) [reply main.go:35]: 🤔

Comment (shogo82148) [reply main.go:35]: > `rand.Int()%len(list)`

this is a typical mistake.
It returns a value drawn from a biased distribution if `MaxInt % len(list) != 0`.

Comment (shmokmt) [reply main.go:35]: I misunderstood the symbols for open and closed sections... 😱 
I'll Revert it. Thanks!

**Pull request merged**